lib/byebug/commands/threads.rb

```diff
@@ -66,6 +58,29 @@ def parse_thread_num_for_cmd(subcmd, arg)
         [c, nil]
       end
     end
+
+    private
+
+    def thread_location(context)
+      backtrace = context.thread.backtrace_locations
+      return nil unless backtrace
+      backtrace[0] if context != Byebug.current_context
+      backtrace
+        .reverse_each
+        .drop_while { |l| context.ignored_file?(l.path) }
+        .take_while { |l| !context.ignored_file?(l.path) }
+        .reverse_each.first
+    end
+
+    def file_line(context, should_show_top_frame)
+      return '' unless should_show_top_frame
+      location = thread_location(context)
+      if location
+        "#{location.path}:#{location.lineno}"
+      else
+        ''
+      end
```

Comment:
For consistency with the rest of the code and with the first line of this method, can you change this to use a guard clause (`return '' unless location`).